airflow/providers/databricks/hooks/databricks.py

```diff
@@ -352,3 +355,50 @@ def get_repo_by_path(self, path: str) -> Optional[str]:
             return str(result['object_id'])
 
         return None
+
+    def import_notebook(self, dbfs_path: str, raw_code: str, language: str, overwrite: bool = True):
```

Comment:
Add a format option - besides `SOURCE`, notebooks could be imported as `HTML`, `DBC`, etc. in this case `language` should be optional.